docs/redirects.asciidoc

```diff
@@ -100,3 +100,15 @@ This content has moved to the <<development-rbac, Security>> page.
 == TSVB
 
 This page was deleted. See <<tsvb>>.
+
+[role="exclude",id="managing-cross-cluster-replication"]
+== Cross-Cluster Replication
+
+This content has moved. See
+{ref}/ccr-getting-started.html[Set up cross-cluster replication].
+
+[role="exclude",id="working-remote-clusters"]
+== Remote clusters
+
+This content has moved. See
+{ref}/ccr-getting-started.html[Set up cross-cluster replication].
```

Comment:
```suggestion
{ref}/{ref}/ccr-getting-started.html[Remote clusters].
```

Reply:
I think that this link should go to the [Remote clusters](https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-remote-clusters.html) page.